stargz/stargz_test.go

```diff
@@ -55,6 +55,16 @@ func TestWriteAndOpen(t *testing.T) {
 				numTOCEntries(0),
 			),
 		},
+		{
+			name: "1dir_1empty_file",
+			in: tarOf(
+				file("bar.txt", ""),
+			),
+			wantNumGz: 3, // file, TOC, footer
+			want: checks(
+				numTOCEntries(1),
```

Comment:
can you add some more checks? That it's of type "file", that it has the right size (0)?

See some of the checks below (lines 77-82) as examples.

Reply:
sure.  I've added some more checks and pushed a new version